<?xml version="1.0"?>
<launch>

	<arg name="dollar" value="$" />
	<arg name="name"/> <!-- Name of the vehicle being launched -->   
  <arg name="config_package_path"  /> <!-- Name of the package where the configuration files are stored -->
  <arg name="folder"/>  <!-- Name of the folder for the configuration files --> 

 <!-- small description about your node -->

 <node pkg="mission_planner" type="mission_planner_node" name="MissionPlannerNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find mission_planner)/config/config_mission_planner.yaml"/>
  <rosparam command="load" file="$(arg dollar)$(arg config_package_path))/config/$(arg folder)/$(arg name)/addons.yaml"/>
</node>

</launch>
